components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -1374,6 +1387,11 @@ impl Animatable for MaxLength {
     }
 }
 
+impl ToAnimatedZero for MaxLength {
+    #[inline]
+    fn to_animated_zero(&self) -> Result<Self, ()> { Err(()) }
```

Comment:
Yeah, it seems to me that we should return Length(Au(0)) here. But,  in the first place, MaxLength does not have get_zero_value(), so I think it's a different issue.  (I also suspect we should return Length(Au(0)) for MozLength)